src/ServiceControl/Bootstrapper.cs

```diff
@@ -71,10 +71,19 @@ public Bootstrapper(ServiceBase host = null, HostArguments hostArguments = null,
 
             configuration.UseSerialization<JsonSerializer>();
 
-            configuration.Transactions()
-                .DisableDistributedTransactions()
-                .DoNotWrapHandlersExecutionInATransactionScope();
-            
+            if (Settings.EnableDtc)
+            {
+                configuration.Transactions()
+                    .EnableDistributedTransactions()
+                    .WrapHandlersExecutionInATransactionScope();
+            }
+            else
+            {
+                configuration.Transactions()
+                    .DisableDistributedTransactions()
+                    .DoNotWrapHandlersExecutionInATransactionScope();
+            }
+
```

Comment:
Is this the default in Core?

```
configuration.Transactions()
   .EnableDistributedTransactions()
   .WrapHandlersExecutionInATransactionScope();
```

Is it enough to disable the DTC if `Settings.EnableDtc` is `false`?